exercises/largest-series-product/instructions.md

```diff
@@ -0,0 +1,59 @@
+# Instructions
+
+Your task is to look for patterns in the long sequence of digits in the encrypted signal.
+
+The technique you're going to use here is called the largest series product.
+
+Let's define a few terms, first.
+
+- **input** - the long sequence of digits that you need to analyze
+- **series** - a sequence of adjacent digits that is contained within the input
+- **span** - how many digits long the series is
+
+To form a series, take digits that are next to each other in the original input.
```

Comment:
In the definition of series we're using `adjacent`. Here we're using a different wording. Was that on purpose?

Reply:
Yeah, `adjacent` is a big word, and the wording here is equivalent. Maybe I should combine them in the definition though, to make it more comprehensible and then use adjacent down here?

Reply:
I would like that!